main/player.py

```diff
@@ -22,20 +23,17 @@ def add_to_backpack(self, item):
     def remove_from_backpack(self, item):
         if item in self.backpack:
             self.backpack.remove(item)
-            print(f"{item} removed from backpack")
-        else:
-            print(f"{item} not found in backpack")
-
-    def expand_backpack(self):
-        self.backpack_size += 1
-        print(f"Backpack size has increased to {self.backpack_size}")
 
-    def gain_experience(self, gained_experience):
-        self.experience += gained_experience
-        if self.experience >= self.experience_needed:
+    def gain_experience(self, sum_of_enemys_stats):
```

Comment:
You could add typing.

Reply:
Can You explain?

Reply:
Typing is a way to comunicate with others what type of argument you do expect. In this case it will look like this: `def gain_experience(self, sum_of_enemys_stats: int)` beacuse expected type of argument `sum_of_enemys_stats`  is intiger.

Reply:
But it weren't a reason to change request , just sugestion.

Reply:
Yea it's nice way to communicate and avoiding mistakes. Thanks for tip! :)